### PR TITLE
fix(components): fix text wrapping issue in buttons

### PR DESCRIPTION
**Pull request by idrissi-hamza**
## What does this PR do?

This pull request resolves an issue where the button text wraps onto the next line in specific screen sizes when the text contains two or more words. By applying the whitespace-nowrap utility class to the button element, the text now remains on a single line, even on screens with limited space. This enhancement ensures a consistent and visually pleasing user experience across various devices :

### Before (Get Started Button)
before

### After  (Get Started Button)
after

This problem arises in other projects utilizing Shadcn, and this pull request addresses and resolves the issue, as demonstrated in the Cal.com project example:

### Before (Set Up Button)
button-before

### After  (Set Up Button)
button-fixed

### CODE REVIEW

File: apps/www/registry/new-york/ui/button.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { cva, type VariantProps } from "class-variance-authority"
 import { cn } from "@/lib/utils"
 
 const buttonVariants = cva(
-  "inline-flex items-center justify-center rounded-md text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring disabled:pointer-events-none disabled:opacity-50",
+  "inline-flex items-center justify-center whitespace-nowrap rounded-md text-sm  font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring disabled:pointer-events-none disabled:opacity-50",
```
Comment (boulhend) [inline apps/www/registry/new-york/ui/button.tsx:8]: Looks great and clean, just remove extra space after `text-sm`

**Review by boulhend — CHANGES_REQUESTED** [review]

File: apps/www/registry/new-york/ui/button.tsx (hunk 1 above)
Comment (idrissi-hamza) [reply apps/www/registry/new-york/ui/button.tsx]: Done thanks

**Comment by shadcn** [comment]
@idrissi-hamza Does the before and after screenshots look the same or am I reviewing this wrong? Thank you.

**Comment by idrissi-hamza** [comment]
> @idrissi-hamza Does the before and after screenshots look the same or am I reviewing this wrong? Thank you.

My bad, I chose the same caption twice; it is fixed now!

**Review by SoyDiego — APPROVED** [review]

**Review by shadcn — APPROVED** [review]
Thank you.

**Pull request merged**